Build error occurred in arm architecture

**Describe the bug**
During  `$./build.sh container arm` or  `$./build.sh container arm64`, I got the following build error.
```
**********************************
 Create Docker container
**********************************

Error: No such container: edge-orchestration
Error: No such image: edge-orchestration:baobab
docker build --tag edge-orchestration:baobab --file /home/t25kim/work/edge-home-orchestration-go/GoMain/Dockerfile --build-arg PLATFORM=arm32v7 .
Sending build context to Docker daemon  301.4MB
Step 1/17 : ARG PLATFORM
Step 2/17 : FROM $PLATFORM/ubuntu:16.04
 ---> 8eedc387cb89
Step 3/17 : ENV TARGET_DIR=/edge-orchestration
 ---> Using cache
 ---> 5f0a61e388ca
Step 4/17 : ENV HTTP_PORT=56001
 ---> Using cache
 ---> b067bb7683b3
Step 5/17 : ENV MDNS_PORT=5353
 ---> Using cache
 ---> 075a5c1d1c28
Step 6/17 : ENV MNEDC_PORT=8000
 ---> Using cache
 ---> c2bc2afb2a0a
Step 7/17 : ENV MNEDC_BROADCAST_PORT=3333
 ---> Using cache
 ---> 5730d8bb2687
Step 8/17 : ENV ZEROCONF_PORT=42425
 ---> Using cache
 ---> b6b101e9d328
Step 9/17 : ENV APP_MAIN_DIR=GoMain
 ---> Using cache
 ---> b6d791035071
Step 10/17 : ENV APP_BIN_DIR=$APP_MAIN_DIR/bin
 ---> Using cache
 ---> e178739df475
Step 11/17 : ENV APP_NAME=edge-orchestration
 ---> Using cache
 ---> e75f9b70dc2c
Step 12/17 : RUN apt update
 ---> Running in 244a90de8ea7
standard_init_linux.go:211: exec user process caused "exec format error"
The command '/bin/sh -c apt update' returned a non-zero code: 1
Makefile:94: recipe for target 'build-container' failed
make: *** [build-container] Error 1
```

**To Reproduce**
Steps to reproduce the behavior:
 `$./build.sh container arm` or  `$./build.sh container arm64`

--- a/GoMain/Dockerfile
+++ b/GoMain/Dockerfile
@@ -15,14 +15,16 @@ ENV ZEROCONF_PORT=42425
 ENV APP_MAIN_DIR=GoMain
 ENV APP_BIN_DIR=$APP_MAIN_DIR/bin
 ENV APP_NAME=edge-orchestration
-
-# install required tools 
-RUN apt update
-RUN apt install -y net-tools iproute2
+ENV APP_QEMU_DIR=$APP_BIN_DIR/qemu
 
 # copy files
 COPY $APP_BIN_DIR/$APP_NAME $APP_MAIN_DIR/run.sh $TARGET_DIR/
+COPY $APP_QEMU_DIR/ /usr/bin/
 RUN mkdir -p $TARGET_DIR/res/
+
+# install required tools
+RUN apt update
+RUN apt install -y net-tools iproute2
 
 # expose ports
 EXPOSE $HTTP_PORT $MDNS_PORT $ZEROCONF_PORT $MNEDC_PORT $MNEDC_BROADCAST_PORT

--- a/build.sh
+++ b/build.sh
@@ -392,6 +392,7 @@ function build_docker_container() {
 
     docker rm -f $DOCKER_IMAGE
     docker rmi -f $DOCKER_IMAGE:$CONTAINER_VERSION
+    mkdir -p $BASE_DIR/GoMain/bin/qemu
     case $1 in
         x86)
             CONTAINER_ARCH="i386"
@@ -401,9 +402,11 @@ function build_docker_container() {
             ;;
         arm)
             CONTAINER_ARCH="arm32v7"
+            cp /usr/bin/qemu-arm-static $BASE_DIR/GoMain/bin/qemu
             ;;
         arm64)
             CONTAINER_ARCH="arm64v8"
+            cp /usr/bin/qemu-aarch64-static $BASE_DIR/GoMain/bin/qemu
             ;;
         *)
             case "$(uname -m)" in

--- a/doc/platforms/hikey960/hikey960.md
+++ b/doc/platforms/hikey960/hikey960.md
@@ -41,6 +41,12 @@ This section provides how to download and run pre-built Docker image without bui
 ---
 
 ## How to build Edge-Orchestration
+
+Prerequisites: install the qemu packages
+```shell
+$ sudo apt-get install qemu binfmt-support qemu-user-static
+```
+
 Run the `./build.sh` script on your PC and specify the build parameters - `container`,  architecture - `arm64`  (in the case of building in protected mode, add `secure`), see examples below:
 ```shell
 $ ./build.sh container arm64

--- a/doc/platforms/raspberry_pi3/raspberry_pi3.md
+++ b/doc/platforms/raspberry_pi3/raspberry_pi3.md
@@ -28,13 +28,19 @@ There are two options for building a edge-orchestration container:
 1. On your PC and downloading the edge-orchestration container image from the `edge-orchestration.tar` archive (recommended).
 2. Build directly on the Raspberry Pi 3 board.
 ### 1. Using your PC
-Run the `./build.sh` script and specify the build parameters - `container`,  architecture - `arm64`  (in the case of building in protected mode, add `secure`), see examples below:
+
+Prerequisites: install the qemu packages
 ```shell
-$ ./build.sh container arm64
+$ sudo apt-get install qemu binfmt-support qemu-user-static
+```
+
+Run the `./build.sh` script and specify the build parameters - `container`,  architecture - `arm`  (in the case of building in protected mode, add `secure`), see examples below:
+```shell
+$ ./build.sh container arm
 ```
 or for protected mode:
 ```shell
-$ ./build.sh container secure arm64
+$ ./build.sh container secure arm
 ```
 the build result will be `edge-orchestration.tar` archive that can be found `GoMain/bin/edge-orchestration.tar`
 

